package.json

```diff
@@ -71,55 +71,56 @@
     "packaging": "jar"
   },
   "dependencies": {
-    "@angular/common": "11.0.5",
-    "@angular/compiler": "11.0.5",
-    "@angular/core": "11.0.5",
-    "@angular/forms": "11.0.5",
-    "@angular/localize": "11.0.5",
-    "@angular/platform-browser": "11.0.5",
-    "@angular/platform-browser-dynamic": "11.0.5",
-    "@angular/router": "11.0.5",
+    "@angular/common": "11.0.6",
+    "@angular/compiler": "11.0.6",
+    "@angular/core": "11.0.6",
+    "@angular/forms": "11.0.6",
+    "@angular/localize": "11.0.6",
+    "@angular/platform-browser": "11.0.6",
+    "@angular/platform-browser-dynamic": "11.0.6",
+    "@angular/router": "11.0.6",
     "@fortawesome/angular-fontawesome": "0.8.1",
     "@fortawesome/fontawesome-svg-core": "1.2.32",
     "@fortawesome/free-solid-svg-icons": "5.15.1",
-    "@ng-bootstrap/ng-bootstrap": "8.0.0",
+    "@ng-bootstrap/ng-bootstrap": "8.0.1",
     "@ngx-translate/core": "13.0.0",
     "@ngx-translate/http-loader": "6.0.0",
     "bootstrap": "4.5.3",
-    "dayjs": "1.9.7",
+    "dayjs": "1.10.2",
     "file-saver": "2.0.5",
     "ng-multiselect-dropdown": "0.3.1",
     "ngx-infinite-scroll": "10.0.1",
-    "ngx-webstorage": "6.0.0",
+    "ngx-webstorage": "7.0.1",
     "rxjs": "6.6.3",
     "swagger-ui-dist": "3.38.0",
-    "tslib": "2.0.3",
+    "tslib": "2.1.0",
     "zone.js": "0.10.3"
   },
   "devDependencies": {
-    "@angular-builders/custom-webpack": "11.0.0-beta.4",
-    "@angular-builders/jest": "11.0.0-beta.4",
-    "@angular-eslint/eslint-plugin": "0.8.0-beta.6",
-    "@angular/cli": "11.0.5",
-    "@angular/compiler-cli": "11.0.5",
-    "@angular/service-worker": "11.0.5",
+    "@angular-builders/custom-webpack": "11.0.0",
+    "@angular-builders/jest": "11.0.0",
+    "@angular-eslint/eslint-plugin": "1.0.0",
+    "@angular/cli": "11.0.6",
+    "@angular/compiler-cli": "11.0.6",
+    "@angular/service-worker": "11.0.6",
     "@openapitools/openapi-generator-cli": "1.0.13-4.3.1",
     "@types/file-saver": "2.0.1",
-    "@types/jest": "26.0.19",
+    "@types/jest": "26.0.20",
     "@types/node": "13.13.4",
-    "@typescript-eslint/eslint-plugin": "4.11.0",
-    "@typescript-eslint/parser": "4.11.0",
+    "@typescript-eslint/eslint-plugin": "4.12.0",
+    "@typescript-eslint/parser": "4.12.0",
     "browser-sync": "2.26.13",
-    "browser-sync-webpack-plugin": "2.2.2",
+    "browser-sync-webpack-plugin": "2.3.0",
     "concurrently": "5.3.0",
     "copy-webpack-plugin": "6.4.1",
-    "cypress": "6.2.0",
-    "eslint": "7.16.0",
+    "cypress": "6.2.1",
+    "eslint": "7.17.0",
     "eslint-config-prettier": "7.1.0",
     "eslint-webpack-plugin": "2.4.1",
     "friendly-errors-webpack-plugin": "1.7.0",
     "generator-jhipster": "6.10.5",
-    "husky": "4.3.6",
+    "generator-jhipster-entity-audit": "3.3.0",
+    "husky": "4.3.7",
```

Comment:
Was this version update discussed with the team or required?
I don't think this updates should be done without the knowledge of the team

Reply:
This was to update to final version of @angular-builders

Reply:
Was this discussed with the team?